README.md

```diff
@@ -571,6 +573,43 @@ tr-pull --auth=./transcend-api-keys.json --resources=consentManager --file=./tra
+
+This command can be helpful if you are looking to:
+
+- Pull resources from your OneTrust account.
+- Migrate your resources from your OneTrust account to Transcend.
+
+#### Authentication
+
+In order to use this command, you will need to generate a OneTrust OAuth Token with scope for accessing the following endpoints:
+
+- [GET /v2/assessments](https://developer.onetrust.com/onetrust/reference/getallassessmentbasicdetailsusingget)
+- [GET /v2/assessments/{assessmentId}/export](https://developer.onetrust.com/onetrust/reference/exportassessmentusingget)
+- [GET /risks/{riskId}](https://developer.onetrust.com/onetrust/reference/getriskusingget)
+
+To learn how to generate the token, see the [OAuth 2.0 Scopes](https://developer.onetrust.com/onetrust/reference/oauth-20-scopes) and [Generate Access Token](https://developer.onetrust.com/onetrust/reference/getoauthtoken) pages.
+
+#### Arguments
+
+| Argument   | Description                                                                                       | Type    | Default     | Required |
+| ---------- | ------------------------------------------------------------------------------------------------- | ------- | ----------- | -------- |
+| auth       | The OAuth access token with the scopes necessary to access the OneTrust Public APIs.              | string  | N/A         | true     |
+| hostname   | The domain of the OneTrust environment from which to pull the resource (e.g. trial.onetrust.com). | string  | N/A         | true     |
+| file       | Path to the file to pull the resource into. Its format must match the fileFormat argument.        | string  | N/A         | true     |
+| fileFormat | The format of the output file. For now, only json is supported.                                   | string  | json        | false    |
+| resource   | The resource to pull from OneTrust. For now, only assessments is supported.                       | string  | assessments | false    |
+| debug      | Whether to print detailed logs in case of error.                                                  | boolean | false       | false    |
+
+#### Usage
+
+```sh
+# Writes out file to ./oneTrustAssessments.json
+tr-pull-ot --auth=$ONE_TRUST_OAUTH_TOKEN --hostname=trial.onetrust.com --file=./oneTrustAssessments.json
```

Comment:
what do you do with the file after its pulled?

Reply:
For now we do nothing, but I am going to create another command that accepts the file path as input and then pushes it to Transcend via the importOneTrustAssessmentForms mutation.

Reply:
got it, maybe use the same command but have a `--dryRun` parameter or something?

Reply:
or have it dump into the transcend.yml shape and use tr-push to push up (would need to add assessment support to that)

Reply:
was thinking opposite, default behavior is to pull from OT and push to transcend. probaly would make sense to rename to `tr-sync-ot`

then if you specify `--dryRun=true` this would only pull from OT and write to file instead of syncing to transcend

Reply:
makes sense! I'll do that.